.github/workflows/codeql.yaml

```diff
@@ -4,6 +4,7 @@ on:
   push:
     branches:
     - main
+    - vihang/codeql
```

Comment:
You probably are already on this, but just a reminder to clean this up before merging.